src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const strWithStyles = sourceString.split(';');
+  const objectCss = {};
+
+  strWithStyles.forEach((rule) => {
```

Comment:
The use of `forEach` here is appropriate as it does not return a new array and is used to modify an existing object.